src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.Common.targets

```diff
@@ -57,4 +57,19 @@ Copyright (c) .NET Foundation. All rights reserved.
   <UsingTask TaskName="NETSdkInformation" AssemblyFile="$(MicrosoftNETBuildTasksAssembly)" />
   <UsingTask TaskName="ShowPreviewMessage" AssemblyFile="$(MicrosoftNETBuildTasksAssembly)" />
 
+  <PropertyGroup Condition="'$(MSBuildRuntimeType)' == 'Full'">
```

Comment:
Consider adding some comment so this is not moved back again in the future unintentionally